scripts/lldb/firestore.py

```diff
@@ -103,13 +111,16 @@ def add_summary(provider, typename, *args):
 
   api = 'firebase::firestore::api::'
   add_summary(DocumentReference_SummaryProvider, api + 'DocumentReference')
+  add_summary(DocumentSnapshot_SummaryProvider, api + 'DocumentSnapshot', '-e')
```

Comment:
What does `-e` flag do?

Reply:
Expands children by default when printing. A `DocumentReference` can straightforwardly be summarized with just the path. `DocumentSnapshot` has more going on, so this makes the children visible by default when printing.